tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/CloudflowStatus.scala

```diff
@@ -47,7 +47,7 @@ object CloudflowStatus {
   }
 
   private def podReady(ps: App.PodStatus) = {
-    ps.status == PodStatus.Running && ps.nrOfContainersReady == ps.nrOfContainers && ps.nrOfContainers > 0
+    (ps.status == PodStatus.Running && ps.nrOfContainersReady == ps.nrOfContainers && ps.nrOfContainers > 0) || (ps.nrOfContainers == 0 && ps.nrOfContainersReady == 0)
```

Comment:
This does not seem right..

Reply:
IIRC the nrOfContainers can be 0 at first. If this is just for flink status when flink is disabled, rather use the "<external>" marker string to decide this.

Reply:
fixed